docs/differences/python_35.rst

```diff
@@ -0,0 +1,172 @@
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+   | Property docstrings are now writable. This is especially useful for collections.namedtuple() docstrings   |               | 
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+   | Circular imports involving relative imports are now supported.                                            |               |
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+
+
+New Modules
+-----------
+
+* `typing <https://docs.python.org/3/whatsnew/3.5.html#typing>`_ 
+
+* `zipzap <https://docs.python.org/3/whatsnew/3.5.html#zipapp>`_
+
+Changes to MicroPython built-in modules
+---------------------------------------
+
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+   | `collections <https://docs.python.org/3/whatsnew/3.5.html#collections>`_                                                  |  +-----------------------------------------------------------------------------------------------------------+---------------+
+   | The *OrderedDict* class is now implemented in C, which makes it 4 to 100 times faster.                    |               |
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+   | *OrderedDict.items()* , *OrderedDict.keys()* , *OrderedDict.values()* views now support reversed()        |               |
+   | iteration.                                                                                                |               |
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+   | The deque class now defines *index()*, *insert()*, and *copy()*, and supports the + and * operators.      |               |
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+   | Docstrings produced by namedtuple() can now be updated.                                                   |               |
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+   | The UserString class now implements the *__getnewargs__()*, *__rmod__()*, *casefold()*, *format_map()*,   |               |
+   | *isprintable()*, and *maketrans()* methods to match the corresponding methods of str.                     |               |
+   +-----------------------------------------------------------------------------------------------------------+---------------+
+   | `heapq <https://docs.python.org/3/whatsnew/3.5.html#heapq>`_  Element comparison in *merge()* can now be  |               |
```

Comment:
please put this heapq heading and URL link in a separate row in the table (same with other entries below)